.changeset/nine-pillows-look.md

```diff
@@ -0,0 +1,5 @@
+---
+'@aws-amplify/ui-react': minor
+---
+
+Add forward ref functionality to MapView
```

Comment:
Minor makes sense here? The `ref` property was not valid prior to this change so that was my reasoning.

Reply:
Right, we publish `ref` on other primitives as minor